# from core.exceptions import CustomException


# class ProductNotFoundException(CustomException):
#     code = 404
#     success = False
#     message = "product not found"
